services/volume-inspector/api/api.go

```diff
@@ -0,0 +1,107 @@
+package api
+
+import (
+	"net/http"
+	"github.com/gorilla/mux"
+	"time"
+	"io/ioutil"
+	"encoding/json"
+	"os"
+	"log"
+)
+
+// Volume folder content
+type VolumeItem struct {
+	Name       string `json:"name"`
+	Size	   int64 `json:"size"`
+	Modified   time.Time `json:"modified"`
+	IsFolder   bool `json:"isFolder"`
+	FolderTree VolumeItems `json:"folderTree"`
+}
+
+type VolumeItems []VolumeItem
+
+type JReply struct {
+	Message string `json:"message"`
+}
+
+type JPost struct {
+	FolderPath string `json:"folderPath"`
+}
+
+func readFolders(currentFolder string, volumeItems VolumeItems) VolumeItems {
+	ifExists, _ := exists(currentFolder)
+	if ifExists == true {
```

Comment:
Unnecessary comparison with true. Please use golint and go vet, it usually checks for these problems.